pkg/quay/quay.go

```diff
@@ -96,6 +96,44 @@ func (c *QuayClient) CreateRepository(r RepositoryRequest) (*Repository, error)
 	return data, nil
 }
 
+// IsRepositoryExists checks if the specified image repository exists in quay.
+func (c *QuayClient) IsRepositoryExists(organization, imageRepository string) (bool, error) {
```

Comment:
why not just `RepositoryExists`?

Reply:
Changed it to `DoesRepositoryExist` now. To make it consistent with other routine name that are starting with a `verb` like `Create` `Delete` `Get`.